app/Http/Controllers/Api/Article/ArticleApiController.php

```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers\Api\Article;
 
+use App\Modules\Articles\Requests\UpdateArticle;
```

Comment:
Sort by namespace length. If you are using VSCode install [PHP Namespace Resolver](https://marketplace.visualstudio.com/items?itemName=MehediDracula.php-namespace-resolver)

Reply:
That will save your time. Please don't forget to read the docs as well.